src/main/java/frc/robot/commands/FeederColorMatcher.java

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.Constants;
+import frc.robot.subsystems.ColorSensor;
+import frc.robot.subsystems.Feeder;
+import edu.wpi.first.wpilibj.Timer;
+
+public class FeederColorMatcher extends Command {
+  private Feeder feeder;
+  private ColorSensor colorSensor;
+  private double startTime;
+  /** Creates a new FeederColorMatcher. */
+  public FeederColorMatcher(Feeder feeder, ColorSensor colorSensor) {
+    this.feeder = feeder;
+    this.colorSensor = colorSensor;
+    addRequirements(feeder);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    feeder.setFeederMotorSpeed(Constants.FEEDER_MOTOR_SPEED);
```

Comment:
The call to set this motor speed needs to be moved to `execute()` so that the motor is moving during the entire time the command is running

Reply:
I think they must have changed the watch dog (or we don't have it enabled) because we're getting away with this behavior. We probably need to enable watchdog to avoid this pattern.